MapboxNavigation/NavigationViewController.swift

```diff
@@ -339,6 +340,11 @@ open class NavigationViewController: UIViewController {
     
     var mapViewController: RouteMapViewController?
     
+    var navigationComponents: [NavigationComponent] {
+        guard let mvc = mapViewController else { return [] }
+        return [mvc]
```

Comment:
Nit: this can be written more succinctly as `return mapViewController ?? []`.

Reply:
It actually can't: 

![image](https://user-images.githubusercontent.com/887225/52092217-13322c00-2574-11e9-9b2a-d064862737cc.png)

Also, this gets overwritten in the subsequent PR.

Reply:
Ah, I missed that it needed to return `mapViewController` in an array.